packages/android_alarm_manager/lib/android_alarm_manager.dart

```diff
@@ -133,6 +133,67 @@ class AndroidAlarmManager {
     return (r == null) ? false : r;
   }
 
+  /// Schedules a one-shot timer to run `callback` at `time`.
+  ///
+  /// The `callback` will run whether or not the main application is running or
+  /// in the foreground. It will run in the Isolate owned by the
+  /// AndroidAlarmManager service.
+  ///
+  /// `callback` must be either a top-level function or a static method from a
+  /// class.
+  ///
+  /// The timer is uniquely identified by `id`. Calling this function again
+  /// again with the same `id` will cancel and replace the existing timer.
+  ///
+  /// If `alarmClock` is passed as `true`, the timer will be created with
+  /// Android's `AlarmManagerCompat.setAlarmClock`.
+  ///
+  /// If `allowWhileIdle` is passed as `true`, the timer will be created with
+  /// Android's `AlarmManagerCompat.setExactAndAllowWhileIdle` or
+  /// `AlarmManagerCompat.setAndAllowWhileIdle`.
+  ///
+  /// If `exact` is passed as `true`, the timer will be created with Android's
+  /// `AlarmManagerCompat.setExact`. When `exact` is `false` (the default), the
+  /// timer will be created with `AlarmManager.set`.
+  ///
+  /// If `wakeup` is passed as `true`, the device will be woken up when the
+  /// alarm fires. If `wakeup` is false (the default), the device will not be
+  /// woken up to service the alarm.
+  ///
+  /// If `rescheduleOnReboot` is passed as `true`, the alarm will be persisted
+  /// across reboots. If `rescheduleOnReboot` is false (the default), the alarm
+  /// will not be rescheduled after a reboot and will not be executed.
+  ///
+  /// Returns a [Future] that resolves to `true` on success and `false` on
+  /// failure.
+  static Future<bool> oneShotAt(
```

Comment:
Since the logic for `oneShot` is a special case of `oneShotAt`, can you change `oneShot` to invoke `oneShotAt` with the time parameter set to `Duration(milliseconds: Datetime.now().millisecondsSinceEpoch + delay.inMilliseconds)`?

Reply:
I think that would be confusing, since if `oneShot` is changed to call `oneShotAt` then it will call `ChannelMethod` with `"Alarm.oneShot"`. I mean from `oneShot` to `oneShotAt` to `"Alarm.oneShot"` would be confusing.
Should I change?

Reply:
@bkonyi waiting for your reply.

Reply:
We can always update `"Alarm.oneShot"` to `"Alarm.oneShotAt"`, but either way we should avoid duplicating logic when we can.

Reply:
Changed `"Alarm.oneShot"` to `"Alarm.oneShotAt"` and also updated `AndroidAlarmManager.oneShot` to use `AndroidAlarmManager.oneShot`.